.travis.yml

```diff
@@ -32,6 +33,10 @@ matrix:
       env: SYMFONY_VERSION="^4.1"
     - php: 7.3 
       env: SYMFONY_VERSION="^5.0" 
+    - php: 8.0
```

Comment:
Oops, it's not ready yet.

Let's try `php: 8.0snapshot` like in Symfony: https://github.com/symfony/symfony/blob/3372b3ef962d3f69ad3180fe186eb5a5f725b11d/.travis.yml#L29